k8s/upgrades/0.7.0-0.8.0/README.md

```diff
@@ -54,7 +54,7 @@ kubectl apply -f https://openebs.github.io/charts/openebs-operator-0.8.0.yaml
 
 - Run `helm repo update` to update local cache with latest package
 - Run `helm ls` to get the release name of openebs. 
-- Upgrade using `helm upgrade -f https://openebs.github.io/charts/helm-values-0.8.0.yaml <release-name> stable/openebs`
+- Upgrade using `helm upgrade --version 0.8.1 <chart-name-from-helm-ls> stable/openebs`
```

Comment:
Why should the version be 0.8.1 when the README is within 0.7.0-0.8.0

Reply:
because the 0.8.1 helm chart points to openebs version 0.8.0.

Reply:
@AmitKumarDas `--version 0.8.1` flag represents the chart version which corresponds to openebs `0.8.0` version.
More details specific to chart and corresponding openebs version can be found here https://hub.kubeapps.com/charts/stable/openebs/0.8.1

@ashishranjan738 please add above link as chart info in README , which will help user to get more details about the version etc.

Reply:
Like @AmitKumarDas pointed out. - The version naming of chart vs openebs version is confusing.

From next time, we will change the helm version convention to something else.  ( could be something like <openebs-version>.c<chart-version>, like 0.8.0. c1, 0.8.0.c2, etc., @prateekpandey14  - can you check for other possible ideas?

Other than above, this PR looks good to merge.